float64 th1
float64 th2
float64 d3
float64 time
string type
---
string status
